manipulation_actions/ChallengeObject challenge_object
---